src/ray/raylet/main.cc

```diff
@@ -37,6 +38,11 @@ int main(int argc, char *argv[]) {
   const std::string java_worker_command = std::string(argv[13]);
   const std::string redis_password = (argc >= 15 ? std::string(argv[14]) : "");
   const std::string temp_dir = (argc >= 16 ? std::string(argv[15]) : "/tmp/ray");
+  const std::string stat_address = (
+      argc >= 17 ? std::string(argv[16]) : "127.0.0.1:8888");
```

Comment:
There's a line `RAY_CHECK(argc >= 14 && argc <= 16);` above that probably needs to be modified.